draft-ietf-quic-transport.md

```diff
@@ -3626,14 +3786,9 @@ change if it is sent multiple times; an endpoint MAY treat receipt of a changed
 octet as a connection error of type PROTOCOL_VIOLATION.
 
 An endpoint MUST NOT send data on any stream without ensuring that it is within
-the data limits set by its peer.  The cryptographic handshake stream, Stream 0,
-is exempt from the connection-level data limits established by MAX_DATA. Data on
-stream 0 other than the initial cryptographic handshake message is still subject
-to stream-level data limits and MAX_STREAM_DATA. This message is exempt from
-flow control because it needs to be sent in a single packet regardless of the
-server's flow control state. This rule applies even for 0-RTT handshakes where
-the remembered value of MAX_STREAM_DATA would not permit sending a full initial
-cryptographic handshake message.
+the data limits set by its peer.  This rule applies even for 0-RTT handshakes
+where the remembered value of MAX_STREAM_DATA would not permit sending a full
+initial cryptographic handshake message.
```

Comment:
CRYPTO_HS frames don't count toward flow control, so this statement is a *non sequitur*.  What does MAX_STREAM_DATA have to do with the handshake message?

Reply:
Just removed the sentence.